Makefile

```
@@ -299,6 +301,11 @@ bats/test: | guard/program/bats
 	cd tests/make && bats -r *.bats
 	@ echo "[$@]: Completed successfully!"
 
+project/validate:
+	@ echo "[$@]: Ensuring the target test folder is not empty"
+	[ "$$(ls -A $(PROJECT_ROOT))" ] && (echo "Project root folder is empty. Please confirm docker has been configured with the correct permissions" && exit 1)
```

do we have a test for this target? i feel like this should be || instead of &&...
```suggestion
	[ "$$(ls -A $(PROJECT_ROOT))" ] || (echo "Project root folder is empty. Please confirm docker has been configured with the correct permissions" && exit 1)
```

---

> do we have a test for this target? 

we do now lol. And agreed on `||` vs `&&`, updated